### PR TITLE
add squeegee recipes

### PR DESCRIPTION
**Pull request by beleafs**
add squeegee recipes

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/squeegee/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{% set version = "0.1.2" %}
+package:
+  name: squeegee
+  version: '{{version}}'
+source:
+  url: 'https://gitlab.com/treangenlab/squeegee/-/archive/v0.1.2/squeegee-v0.1.2.tar.gz'
```
Comment (fanninpm) [inline recipes/squeegee/meta.yaml:6]: To facilitate future updates, try this:
```suggestion
  url: 'https://gitlab.com/treangenlab/squeegee/-/archive/v{{ version }}/squeegee-v{{ version }}.tar.gz'
```

**Comment by fanninpm** [comment]
@BiocondaBot please update

**Comment by fanninpm** [comment]
@BiocondaBot please rebuild

**Comment by fanninpm** [comment]
@BiocondaBot rebuild

**Comment by beleafs** [comment]
I'll update it tonight.

File: recipes/squeegee/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{% set version = "0.1.2" %}
+package:
+  name: squeegee
+  version: '{{version}}'
+source:
+  url: 'https://gitlab.com/treangenlab/squeegee/-/archive/v{{ version }}/squeegee-v{{ version }}.tar.gz'
+  sha256: 012722b7354a1b0b554ab3b9551c981b402398207b225b4dc20a9a6cb0682d4d
```
Comment (fanninpm) [inline recipes/squeegee/meta.yaml:7]: Try running `curl https://gitlab.com/treangenlab/squeegee/-/archive/v0.1.2/squeegee-v0.1.2.tar.gz | shasum -a 256` to get the correct hash.

File: recipes/squeegee/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    - numpy
+    - mash
+    - meryl
+test:
+  imports:
+    - Squeegee
+about:
+  home: https://gitlab.com/treangenlab/squeegee
+  license: MIT
+  license_file: LICENSE
```
Comment (fanninpm) [inline recipes/squeegee/meta.yaml:27]:
```suggestion
```

**Review by bkille — APPROVED** [review]

**Comment by bkille** [comment]
@BiocondaBot please add label

**Comment by beleafs** [comment]
@bioconda please merge

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**